test-expected/sequelize/db-analysis-output/default_values.expected.js

```diff
@@ -0,0 +1,148 @@
+const Sequelize = require('sequelize');
```

Comment:
[MISC] Can you tell me what change in here 🙏

Reply:
When reading how the test suite work, you'll see that the files in `test-expected/db-analysis-output` are used both as:
- Expected output (as their name imply) to check that the `database-analyzer` is doing its job. This happens in `test/services/database-analyzer/...`
- Input for all the `dumper` tests. This happens in `test/services/dumper/dumper-sequelize.test.js`

So it goes like this:
```
test-fixtures/postgres/addresses.sql
test-fixtures/mysql/addresses.sql      ---analyzer---> test-expected/sequelize/db-analysis-output/addresses.json ---dumper--> test-expected/sequelize/dumper-output/addresses.expected.js
test-fixtures/mssql/addresses.sql
```

This was done for all tests besides the one for default values: there is no corresponding SQL files for the "expected analysis output" file which was on the test-expected folder.

The extraction of default values from the database was left completely untested.
What was tested was only the second step (dumping).

That's why I deleted and completely rewrote this file (and lost some of the tests in the way): I wrote a SQL file so that I get to test the db-analyzer and needed the expected-ouput file to match it.

**As a matter of fact, the test coverage on this feature is better than before this PR, but it is still strongly lacking, as the default values extraction is still untested for mssql and mysql.**